pymc3/gp/cov.py

```diff
@@ -35,26 +37,29 @@ def __init__(self, input_dim, active_dims=None):
             if len(active_dims) != input_dim:
                 raise ValueError("Length of active_dims must match input_dim")
 
-    def __call__(self, X, Z=None, diag=False):
+    def __call__(self, X, Xs=None, diag=False):
```

Comment:
Z seemed most commonly used to represent inducing point locations in the lit.  I changed it to Xs to look more like $X_*$